app/Resources/views/card/new.html.twig

```diff
@@ -4,93 +4,140 @@
-                    </div>
-                </div>
-            </div>
-            <div>
-                <label for="appbundle_card" class="required">Title</label>
-                <input type="text" id="appbundle_card" name="appbundle_card[title]" required="required" maxlength="255">
-            </div>
-            <div>
-                <label for="appbundle_card" class="required">Description</label><textarea id="appbundle_card" name="appbundle_card[description]" required="required"></textarea>
-            </div>
-            <div>
-                <label for="appbundle_card_project" class="required">Project</label>
-                {% for project in projects %}
-                    <input type="radio" value="{{project.name}}" name="appbundle_card[project]"
-                    {% if project.name == card.project %}checked="checked"{% endif %}> {{project.name}}
-                {% endfor %}
+
+      <div class="form-row">
+          <div class="form-group col-md-3">
+              <label class="required">Datetime</label>
+              <div id="appbundle_card_datetime">
+                  <div id="appbundle_card_datetime_date">
+                      <select id="appbundle_card_datetime_date_year" name="appbundle_card[datetime][date][year]">
+                      {% for year in range("now"|date('Y'), ("now"|date('Y')) + 1) %}
+                          {% if "now"|date('Y') == year %}
+                              <option selected="selected" value="{{year}}">{{year}}</option>
+                          {% else %}
+                              <option value="{{year}}">{{year}}</option>
+                          {% endif %}
+                      {% endfor %}
+                      </select>
+                      /
+                      <select id="appbundle_card_datetime_date_month" name="appbundle_card[datetime][date][month]">
+                      {% for month in range(1, 12) %}
+                          {% if "now"|date('m') == month %}
+                              <option selected="selected" value="{{month}}">{{month}}</option>
```

Comment:
@Fukotaku is this?

Reply:
Oups, it's for edit, not new. Sorry ^^'

Reply:
I just add "/" for separation yyyy/mm/dd